src/module-config/module-config.ts

```diff
@@ -29,10 +29,14 @@ export async function getConfig(moduleName: string): Promise<any> {
     }
 
     if (importMapConfigExists) {
-      await System.import("config-file").then(res => {
-        configs.unshift(res.default);
-        importMapConfigHasBeenAdded = true;
-      });
+      try {
+        await System.import("config-file").then(res => {
```

Comment:
This syntax is pretty unusual - `await` is an alternative to `.then()`. Here's the more common way to do this:

```ts
const config = await System.import('config-file')
configs.unshift(res.default)
```